src/main/primary_entry/css/page/map/MapView.css

```diff
@@ -63,7 +63,7 @@ Markers
 .cluster-marker.permit { background-color: rgba(0, 64, 255, 0.7); }
 .cluster-marker.closed-temp { background-color: rgba(160, 160, 160, 0.7); }
 .cluster-marker.closed-perm { background-color: rgba(0, 0, 0, 0.7); }
-.cluster-marker.user { background-color: rgba(0, 192, 0, 0.7); }
+.cluster-marker.user { background-color: rgba(0, 224, 0, 0.7); color: black; }
```

Comment:
Made cluster/tooltip text for custom markers black (and added a white pin to the marker itself) to accommodate red-green colorblindness